### DISCUSSION TITLE
hELLO 버그사항 및 건의사항

### DISCUSSION THREAD

**Discussion by Sooho-Kim** [pronist/hELLO#89] [Q&A] [answered]
안녕하세요! 티스토리 스킨 감사히 잘 사용하고 있습니다.
사용하면서 버그인지는 모르겠습니다만 수정될 수 있는 지 확인해주시면 감사하겠습니다.
1. bullet point가 최초 작성할 때에는 탭으로 문단구분을 해줬으나, 적용이 되지않습니다.

- 글쓰는 화면
![화면 캡처 2021-01-29 224238](https://user-images.githubusercontent.com/67626878/106282549-173cd580-6284-11eb-9a6a-c63575f12fef.png)
- 글 작성 후 보여지는 화면
![화면 캡처 2021-01-29 224218](https://user-images.githubusercontent.com/67626878/106282547-16a43f00-6284-11eb-8057-ce433b5461d8.png)

2. 모바일(다크모드)에서 글을 확인하다보면 중간 중간 글 일부가 거의 회색으로 변경되어 있는 경우가 있습니다.
![KakaoTalk_20210129_224124828_01](https://user-images.githubusercontent.com/67626878/106282791-70a50480-6284-11eb-98d6-68471fb090e0.jpg)

3. 이것은 티스토리의 문제인 것 같습니다만 혹시 해결방법이 있을까해서 여쭤봅니다. Latex를 적용한 화면을 핸드폰으로 웹화면으로 확인할 때, Latex 길이 때문에 아래와 같은 현상이 발생합니다. Latex가 없거나 짧으면 크게 문제가 되지는 않습니다만 잘라주는 게 최선이겠죠?ㅠㅠ
![KakaoTalk_20210129_224124828_02](https://user-images.githubusercontent.com/67626878/106282971-ac3fce80-6284-11eb-98b3-589eb47ceef1.jpg)

항상 감사합니다!

**Comment by pronist** [comment]
안녕하세요. 

1번, 일단 제가 신경쓰지 못한 부분이므로 해결해야 할 것 같습니다. 

2번, 다크모드라고 할지라도 색상을 완전히 강제하지 않습니다. 해당 글의 HTML 을 여셔서 인라인 스타일 색상이 입혀있을 겁니다. 그것을 제거하면 문제가 해결됩니다. 티스토리 에디터에서 복사 붙여넣기를 하면 자동으로 인라인 속성이 부여되고, 에디터 자체의 기능으로 인해 색상이 강제됩니다. 그렇다고 다크모드에서 색상을 강제하려니 다른 문제가 발생하더군요 :(

3번은 ... 라텍스의 글자가 화면 범위를 넘어가서 그런 듯싶습니다. 이 문제는 스킨 자체의 문제라기 보단, 라텍스가 다음 라인으로 넘어가지 않고 하나의 라인으로 처리하려는 속성이 있어 그런것으로 추측됩니다. 해결 방안이라기보단 시도해볼 수 있는 문제는 라텍스에 적용되는 CSS를 건들여보는 것이군요.

**Comment by Sooho-Kim** [comment]
답변 감사드립니다! 감사합니다!

**Comment by pronist** [comment, marked as answer]
1번은 해결되었습니다. 2번은 그렇다쳐도, 3번같은 경우에는 사실 스킨 아닌 외부에서 적용되는 기능에 대한 사항이라 공식 패치에는 적용되기 어려울 수도 있다는 점 양해부탁드립니다!

**Comment by Sooho-Kim** [comment]
감사합니다!!

**Comment by pronist** [comment]
@m1nnh 스킨의 최신버전을 받으셔서 새로 적용시키시면 된답니다.

**Comment by pronist** [comment]
@m1nnh 거기에서 받으신 버전이 최신버전입니다. ```index.xml``` 파일의 버전이 ```3.3.0``` 으로 되어있나 보시면 됩니다.  만약 그게 아니라면 원하는 것이 다른 것일지도 모릅니다. 원하시는 것과 실제 적용된 사진을 올려주세요.

**Comment by pronist** [comment]
@m1nnh 글 주소 주세요. 직접 확인해봐야 알 것 같네요.
